cmd/contour/serve.go

```diff
@@ -300,10 +300,9 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	// managing the close process when the other goroutines are closed.
 	g.AddContext(func(electionCtx context.Context) {
 		log := log.WithField("context", "leaderelection")
-		if !ctx.EnableLeaderElection {
+		if ctx.DisableLeaderElection {
```

Comment:
Please add a TODO that we can avoid setting up leadership election if its disable; just close leader OK and don't call AddContext.

Reply:
Same as line 312 which I can't add a comment on because github.